src/main/java/org/kainos/ea/resources/EmployeeController.java

```diff
@@ -0,0 +1,26 @@
+package org.kainos.ea.resources;
+
+import io.swagger.annotations.Api;
+import org.kainos.ea.client.FailedtoCreateEmployee;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+public class EmployeeController {
+
+    @Api("Evian Week")
```

Comment:
Is this a sensible name for the API?

Reply:
Resolved Issue and Added in our Delivery Controllers and added all to apploication file